src/w3d/renderer/dx8wrapper.h

```diff
@@ -197,15 +206,15 @@ class DX8Wrapper
     static void Clamp_Color(Vector4 &color);
     static unsigned int Convert_Color_Clamp(const Vector4 &color);
     static void Set_DX8_Light(int index, D3DLIGHT8 *light);
-    static void Get_DX8_Transform(D3DTRANSFORMSTATETYPE transform, Matrix4 &m);
-    static void Set_DX8_Transform(D3DTRANSFORMSTATETYPE transform, const Matrix4 &m);
+    static void Get_DX8_Transform(D3DTRANSFORMSTATETYPE transform, D3DMATRIX &m);
+    static void Set_DX8_Transform(D3DTRANSFORMSTATETYPE transform, const D3DMATRIX &m);
```

Comment:
Matrix4 has been changed to D3DMATRIX for these 2 function because they clearly expect matrix in Direct X coordinate space, not a transposed Matrix4.